source/game.programme.programmedata.specials.bmx

```diff
@@ -342,6 +342,7 @@ Type TSportsProgrammeData Extends TProgrammeData {_exposeToLua}
 
 		Local league:TNewsEventSportLeague = team.GetLeague()
 		If league
+'		-> %teamXrank% umbenennen in ${.self:sports:team... ?
```

Comment:
Ist ein Ersetzen in den Resourcedateien unproblematisch? Die Ersetzung muss ja im Kontext des Nachrichtenevents erfolgen, d.h. Teams und Namen müssen ohne Angabe einer festen ID im Text korrekt aufgelöst werden können.
Dasselbe gilt vermutlich auch für Sportprogramme!?